flake.nix

```diff
@@ -0,0 +1,33 @@
+{
+  description = "Dependencies for development";
+
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+  };
+
+  outputs = { self, nixpkgs, flake-utils } @ imports: (
+    flake-utils.lib.eachDefaultSystem (
+      system: (
+        let
+          pkgs = import nixpkgs { inherit system; };
+          baseDependencies = with pkgs; [
+            python3
+          ];
+        in
+        {
+          formatter = pkgs.nixpkgs-fmt;
+          devShells = rec {
+            default = docs;
+            docs = pkgs.mkShell {
+              buildInputs = with pkgs; [
+                nodejs
+              ]
```

Comment:
Should this also contain Rust?

Reply:
Using the Rust from nixpkgs instead of from oxalica's overlay.  
Rationale: We probably don't need the newest Rust stable (or, god forbid, nightly), and the oxalica download itself is already 38MB, which is quite huge.

(see https://github.com/oxalica/rust-overlay/pull/172)

Reply:
Makes sense, but I get a linker error now on my Macbook:
>   = note: ld: library not found for -liconv
          clang-16: error: linker command failed with exit code 1 (use -v to see invocation)

Reply:
Looks like I'm running into this: https://github.com/NixOS/nixpkgs/issues/206242
https://github.com/oxalica/rust-overlay/blob/c9a793a5278f711a59fe77b9bf54b215667022c6/lib/mk-aggregated.nix#L29-L30